CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```

Comment:
Should replace this with our `CONTRIBUTING.md` doc? looks like it is an artifact from a google project?

Reply:
yep, I think you're right ... I must have git-added it after removing the cached node-modules. I'll remove!